### PR TITLE
Prominent links with Grid

### PR DESCRIPTION
**Pull request by ikesau**
A rewrite of the Prominent Link CSS to get it to play nice with our new columns.

[Staging demo](https://ptolemy.owid.cloud/admin/posts/preview/50492)

~@mlbrgl could you check these out for usecases I may have missed? I'm not sure what the lightbox interaction is, for example.~ Done.

Screenshot 2022-09-29 at 12-32-50 Prominent links demo

### CODE REVIEW

**Comment by ikesau** [comment]
I think I'll slightly modify this to introduce a grid mixin that makes it easier to define subgrids. It's going to be annoying to constantly have to write out the template each time.

Also need to handle other column layouts, the current rules won't align if someone puts a prominent link in the right-hand side of a sticky-left column, for example.

**Comment by mlbrgl** [comment]
Sounds good! 

Just so you know, there is a suite of cypress tests for prominent links. You can run it with `yarn testCypress` and then select "kitchenSink.js"

Screenshot 2022-09-29 at 13 32 36

There shouldn't be any lightbox popping up when clicking a prominent link.

**Comment by ikesau** [comment]
Cool, seems like everything works then. 👍 

After a discussion with Marwa we decided to change the links back to having padding instead of an outline,  because otherwise the boxes were entirely flush with sticky graphers in the other column.

So, a whole refactor for what could have basically just been a one-line change in the padding. Alas! 😅

File: devTools/cypress/integration/kitchenSink.js [hunk 1]
```diff
@@ -159,11 +159,12 @@ describe("Kitchen sink", function () {
...
-            title: "Adjusted net savings per capita",
-            src: "http://localhost:3030/grapher/exports/adjusted-net-savings-per-person.svg",
-        })
+        // N/A
+        // testProminentLink({
+        //     testTitle:
+        //         "[NOT RECOMMENDED] Grapher link redirected to grapher (in wordpress)",
+        //     title: "Adjusted net savings per capita",
+        //     src: "http://localhost:3030/grapher/exports/adjusted-net-savings-per-person.svg",
+        // })
```
Comment (mlbrgl) [inline devTools/cypress/integration/kitchenSink.js:168]: does this test need updating?

Comment (ikesau) [reply devTools/cypress/integration/kitchenSink.js:168]: Seems so, yeah. I no longer have `adjusted-net-savings-per-person` in my exports after running a local build. We should probably try to not have these tests rely on the state of someone's local development DB.

Comment (mlbrgl) [reply devTools/cypress/integration/kitchenSink.js:168]: Yes could you please revert this so I'll take a look next time I run this and update the test?

**Review by mlbrgl — CHANGES_REQUESTED** [review]
https://github.com/owid/owid-grapher/pull/1684/files#r987658657

**Review by mlbrgl — APPROVED** [review]

**Pull request merged**